### ISSUE TITLE
super doesn't work in an arrow function

### ISSUE THREAD

**Issue by aexmachina** [6to5/6to5#688]
The following code throws an exception `Cannot read property 'topLevelThisReference' of undefined`:

```js
class Test extends Foo {
  fails() {
    ['a'].map(val => super.foo(val));
  }
}
class Foo {
  foo(val) {}
}
```

The stack trace is:

```
TypeError: /Users/simonwade/dev/Workspace/franq/test.js: Cannot read property 'topLevelThisReference' of undefined
    at ReplaceSupers.getThisReference (/Users/simonwade/dev/Workspace/franq/node_modules/6to5/lib/6to5/transformation/helpers/replace-supers.js:178:11)
    at ReplaceSupers.specHandle (/Users/simonwade/dev/Workspace/franq/node_modules/6to5/lib/6to5/transformation/helpers/replace-supers.js:271:19)
    at visitor.enter (/Users/simonwade/dev/Workspace/franq/node_modules/6to5/lib/6to5/transformation/helpers/replace-supers.js:157:21)
    at TraversalContext.enterNode (/Users/simonwade/dev/Workspace/franq/node_modules/6to5/lib/6to5/traverse/index.js:67:16)
    at TraversalContext.visitNode (/Users/simonwade/dev/Workspace/franq/node_modules/6to5/lib/6to5/traverse/index.js:119:15)
    at TraversalContext.visit (/Users/simonwade/dev/Workspace/franq/node_modules/6to5/lib/6to5/traverse/index.js:144:17)
    at traverseNode (/Users/simonwade/dev/Workspace/franq/node_modules/6to5/lib/6to5/traverse/index.js:173:17)
    at TraversalContext.visitNode (/Users/simonwade/dev/Workspace/franq/node_modules/6to5/lib/6to5/traverse/index.js:132:5)
    at TraversalContext.visit (/Users/simonwade/dev/Workspace/franq/node_modules/6to5/lib/6to5/traverse/index.js:152:26)
    at traverseNode (/Users/simonwade/dev/Workspace/franq/node_modules/6to5/lib/6to5/traverse/index.js:173:17)
```

**Comment by aexmachina** [comment]
Thanks @sebmck, any idea when this will be released?

**Comment by sebmck** [comment]
@aexmachina Few minutes.

**Comment by sebmck** [comment]
Fixed as of 3.3.9, thanks!
